docs/docs/userguide/configuration/CONFLUENCE_CONFIGURATION.md

```diff
@@ -18,6 +18,24 @@ For more information, see [Confluence Version Tags](https://hub.docker.com/r/atl
 confluence_version_tag = "<CONFLUENCE_VERSION_TAG>"
 ```
 
+### Number of Confluence application nodes
+
+The initial Confluence installation require to be started only with a single application node. After all the setup steps
```

Comment:
`The initial Confluence installation need's to be started with a single application node.`